aws-xray-auto-instrumentation-agent-runtime/src/main/java/com/amazonaws/xray/agent/AgentRuntimeLoader.java

```diff
@@ -1,23 +1,23 @@
 package com.amazonaws.xray.agent;
 
-import com.amazonaws.xray.agent.handlers.downstream.AWSV2Handler;
-import com.amazonaws.xray.contexts.LambdaSegmentContextResolver;
-import software.amazon.disco.agent.AgentRuntimeLoaderInterface;
 import software.amazon.disco.agent.event.EventBus;
-import software.amazon.disco.agent.logging.LogManager;
-import software.amazon.disco.agent.logging.Logger;
 import com.amazonaws.xray.agent.config.XRaySDKConfiguration;
 import com.amazonaws.xray.agent.dispatcher.EventDispatcher;
 import com.amazonaws.xray.agent.handlers.downstream.AWSHandler;
+import com.amazonaws.xray.agent.handlers.downstream.AWSV2Handler;
 import com.amazonaws.xray.agent.handlers.downstream.HttpClientHandler;
 import com.amazonaws.xray.agent.handlers.upstream.ServletHandler;
 import com.amazonaws.xray.agent.listeners.XRayListener;
 import com.amazonaws.xray.agent.models.XRayTransactionContextResolver;
 import com.amazonaws.xray.agent.models.XRayTransactionState;
+import com.amazonaws.xray.contexts.LambdaSegmentContextResolver;
 import com.amazonaws.xray.contexts.SegmentContextResolverChain;
 import com.amazonaws.xray.strategy.DefaultContextMissingStrategy;
 import com.amazonaws.xray.strategy.sampling.CentralizedSamplingStrategy;
 
+import org.apache.commons.logging.Log;
```

Comment:
Is it possible to use slf4j instead of apache for these logs?

Reply:
We use Apache logs in the X-Ray Java SDK since it is provided transitively by the AWS SDK. I wanted to use the same library here so we could provide customers the same [logging configuration](https://docs.aws.amazon.com/xray/latest/devguide/xray-sdk-java-configuration.html#xray-sdk-java-configuration-logging) experience. However, once we remove the AWS SDK dependency I'd be open to change both the SDK and the agent (along with configuration experience) to SLF4J. What do you think?

Reply:
Thanks for the context, SGTM